### DISCUSSION TITLE
default at startup

### DISCUSSION THREAD

**Discussion by ImPhoenix555** [Rem0o/FanControl.Releases#1373] [Q&A]
how do I set one of my configurations to start when my PC turns on instead of another? currently, every time I turn my PC on it starts on a config that I don't want it to start on, so how do I change the default?

**Comment by Rem0o** [comment]
The last config you saved or exited the app with should be the one used next time the app is started.

**Reply by ImPhoenix555** [reply]
it might be because I was changing the config in the system tray instead of through the app. thanks for the reply.
